__e2e__/root.test.ts

```diff
@@ -0,0 +1,64 @@
+import path from 'path';
+
+import {
+  spawnScript,
+  runCli,
+  getTempDirectory,
+  cleanup,
+  writeFiles,
+} from '../jest/helpers';
+
+const cwd = getTempDirectory('test_different_roots');
+
+// Register all packages to be linked
+beforeAll(() => {
+  for (const pkg of ['platform-ios', 'platform-android']) {
+    spawnScript('yarn', ['link'], {
+      cwd: path.join(__dirname, `../packages/${pkg}`),
+    });
+  }
+});
+
+beforeEach(() => {
+  // Clean up folder and re-create a new project
+  cleanup(cwd);
+  writeFiles(cwd, {});
+
+  // Initialise React Native project
+  runCli(cwd, ['init', 'TestProject']);
```

Comment:
Let's move init to beforeAll, so it get downloaded only once. We don't really modify files, so there's no need to cleanup between tests.

Reply:
Done